patternfly/patternfly-elements
# Fix pfe-icon stretch + working colors in Edge

## Fix pfe-icon stretch

<img width="393" alt="Screen Shot 2019-10-28 at 4 05 25 PM" src="https://user-images.githubusercontent.com/1840295/67714286-fbba8f00-f99d-11e9-981b-37c452c3fad1.png">

---

### What has changed and why

* Still seeing an issue with the icon stretching in certain layouts so added a `width: fit-content;` attribute to limit the width.

### Testing instructions
Be sure to include detailed instructions on how your update can be tested by another developer.

1. Check out the latest code and preview pfe-icon with circled background in a layout trying to stretch the element.  It should honor it's selected sizing and aspect ratio.

<img width="1384" alt="Screen Shot 2019-10-28 at 4 37 51 PM" src="https://user-images.githubusercontent.com/1840295/67716020-5b666980-f9a1-11e9-9aad-e332958eabd6.png">


#### Browser requirements
Your component should work in all of the following environments:


### Ready-for-merge Checklist



**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
